docker/dockerfile.ctr

```
@@ -44,9 +44,11 @@ RUN apt update -y --fix-missing && \
       add-apt-repository -y ppa:deadsnakes/ppa && \
       apt update -y --fix-missing
 
+RUN pip install git+git://github.com/gevent/gevent.git#egg=gevent
```

Nice! I like this way of installing from source.

Rather than installing the latest code from master though, can we pin to the last stable release ?

```suggestion
RUN pip install git+git://github.com/gevent/gevent.git@21.8.0#egg=gevent
```